src/LayerEditor/ui/panels/surfaces.py

```diff
@@ -591,7 +487,7 @@ def _check_file_path(self, file):
         If not try copy it.
         Returns new path or empty string if it is not possible.
         """
-        layer_file_dir = os.path.dirname(cfg.curr_file)
+        layer_file_dir = os.path.dirname(self.le_model.curr_file)
```

Comment:
E.g. main_file_dir may be part of the file manager.

Reply:
There is not support for multiple files, therefore file manager does not make sense.

Reply:
make a lambda function in the constructor.